src/components/entity-info.component.js

```diff
@@ -143,6 +143,22 @@ export const EntityInfo = ({ entity, orgs, language, navigation }: Props) => {
           onPress={() => Communications.web(getBlogLink(entity.blog))}
           underlayColor={colors.greyLight}
         />}
+
+      {!!entity.webSite &&
```

Comment:
The block above should be removed once the new API is implemented on Users.
For now I duplicated the block instead of putting 3 `entity.web_site ? entity.web_site : entity.webSite` in the existing block

Reply:
@lex111 left you some more comments on gitter to avoid writing another page here :)